src/Compilers/CSharp/Portable/Utilities/ValueSetFactory.ByteTC.cs

```diff
@@ -0,0 +1,61 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Diagnostics;
+
+#nullable enable
+
+namespace Microsoft.CodeAnalysis.CSharp
+{
+    using static BinaryOperatorKind;
+
+    internal static partial class ValueSetFactory
+    {
+        private struct ByteTC : NumericTC<byte>
```

Comment:
>ByteTC [](start = 23, length = 6)

Similarly, I'm unsure where we get the coverage for `ByteTC`. There are only two assertions related to `ForByte` in `ValueSetTests.cs`

Reply:
The type class is exhaustively exercised indirectly by those two tests.  These basic operations are needed to construct the result of the Related operation.

---
In reply to: [381024151](https://github.com/dotnet/roslyn/pull/41503#discussion_r381024151) [](ancestors = 381024151)